### DISCUSSION TITLE
Playback and Content-Range

### DISCUSSION THREAD

**Discussion by stardast** [bluenviron/mediamtx#3221] [Questions]
### Question

Hi, I'm using mediamtx 1.6.0 to stream live video from an rtsp camera. I would like to record the video stream and offer a playback functionality. If I follow the instructions in the readme:
<video controls>
   <source src="http://localhost:9996/get?path=[mypath]&start=[start_date]&duration=[duration]" type="video/mp4" />
</video>
I view the video but it is not possible to move within it, I think because the Content-Range and Content-Length headers are missing.
Is something missing in my configuration or is it an unsupported feature?

Thank you,
Paolo

**Comment by stardast** [comment]
related to this [issue](https://github.com/bluenviron/mediamtx/discussions/3045)

**Comment by aler9** [comment]
> I view the video but it is not possible to move within it

This is natively unsupported but is really easy to implement: add a "on click" handler on the seeking bar and use it to change the starting timestamp of the video requested to the playback server.

[a comment by an automated account is not shown]
